src/containers/forms/CreateCategoryForm.jsx

```diff
@@ -92,6 +105,7 @@ class CreateCategoryForm extends Component {
       errors: validateFormFields(category),
     }, () => {
       if (Object.keys(this.state.errors).length === 0) {
+        category.supports_multiple = supportsMultiple;
```

Comment:
Let us add the key-value pair for `supports_multiple` while creating the `category constant`.

Reply:
I wanted to do that but it is not a good idea because then it has to conform to the camelCase convention. Also that means the `validateFormFields` function will try to validate it and it seems that function is designed to validate string values. This means it throws an error when it attempts to trim a boolean value